src/Microsoft.ML.Data/Scorers/RowToRowScorerBase.cs

```diff
@@ -49,7 +49,7 @@ private protected RowToRowScorerBase(IHost host, ModelLoadContext ctx, IDataView
             ctx.LoadModel<ISchemaBindableMapper, SignatureLoadModel>(host, out Bindable, "SchemaBindableMapper");
         }
 
-        public sealed override void Save(ModelSaveContext ctx)
+        private protected sealed override void SaveModel(ModelSaveContext ctx)
```

Comment:
>private protected sealed override [](start = 8, length = 33)

do you need all this modifiers?

Reply:
I think so,  sealed override means that derived classes cannot override this method. Since it is a base class, I think it could be fair that we want to tell developers deriving from this class that this should not be overwritten. 

the change from public to private protected comes from the fact that the base class is public, however, I agree that maybe in the future that could not be useful anymore. For the time being I would keep it like this.

---
In reply to: [255239221](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255239221) [](ancestors = 255239221)